YLTableViewExample/Podfile.lock

```diff
@@ -1,5 +1,5 @@
 PODS:
-  - YLTableView (2.0.0)
+  - YLTableView (2.1.4)
```

Comment:
this is the old version - rerun `pod install` to make sure it's up-to-date with the pod version?

Reply:
Good catch. Pushed update.